docs/source-app/examples/github_repo_runner/.lightning

```diff
@@ -0,0 +1 @@
+name: github_repo_runner
```

Comment:
Should we have it in the examples?

Reply:
Yes, we have been committing .lightning file.